drafts/25-xxx-specifications.txt

```diff
@@ -99,68 +98,107 @@ li32. Fortran source fragments may be continued with a continuation
 
 
 
-2.1.2 Case sensitivity of identifiers
--------------------------------------
+2.2 Case sensitivity of identifiers
+-----------------------------------
 
 cs01. Directive names are case-sensitive and recognized in lower-case.
 
-cs02. Macro names and function-like macro argument names are
+cs03. Macro names and function-like macro argument names are
       case-sensitive.
 
-cs03. Preprocessor-defined macro names are case-sensitive.
+cs05. Preprocessor-defined macro names are case-sensitive.
 
 
 
-2.1.3 Significance of whitespace
---------------------------------
+2.3 Significance of whitespace
+------------------------------
 
 ws01. The whitespace characters blank and horizontal tab character may appear
       on directive lines.
 
 ws03. Whitespace characters are significant in determining token
       boundaries in preprocessor directive lines.
 
-ws02. Whitespace characters are significant in determining token
+ws05. Outside of character constants, multiple whitespace characters
+      on a directive line are treated as a single space.
+
+ws07. Whitespace characters are significant in determining token
       boundaries for the purposes of recognizing macro names,
       in both fixed-form and free-form Fortran source fragments.
 
-ws03. Whitespace characters are significant in determining token
+ws09. Whitespace characters are significant in determining token
       boundaries for the purposes of recognizing macro names,
       in both fixed-form and free-form Fortran comment lines.
 
-ws04. In fixed-form input, macro names are not recognized as such when
+ws11. In fixed-form input, macro names are not recognized as such when
       spaces are inserted into their invocations.
 
 
-2.2 Comments
+2.4 Comments
 ------------
 
 co01. Directive lines may contain C-style '/*' ... '*/' comments.
 
-co02. Directive lines shall not contain C style '//' comments.
+co03. Directive lines shall not contain C style '//' comments.
 
-co02. '/*' ... '*/' comments on directive lines shall extend past a
+co05. '/*' ... '*/' comments on directive lines shall extend past a
       new-line only if the line ends in '\' new-line, indicating a
       continuation line. (Maybe controversial)
```

Comment:
I think we can remove the non-normative commentary here.

Everything is potentially controversial.

```suggestion
      continuation line.
```